### PR TITLE
post, share articles

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/PersonalApplications.rst [hunk 1]
```diff
@@ -936,6 +936,7 @@ In this section, the following notification streams will be presented:
 * :ref:`Post on your activity stream <PostOnYourActivityStream>`
 * :ref:`Post in your space <PostInSpace>`
 * :ref:`Task Management activities <TaskManagement>`
+* :ref:`News activities notifications <NewsNotifications>`
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
* :ref:`News notifications <NewsNotifications>`
```

File: docs/PersonalApplications.rst [hunk 2]
```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
    Click on the notification stream, you will be taken to the detailed
    task pane in the Task Management application.
 
+.. _NewsNotifications :
+
+
+**News post in a space**
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
**News posted in a space**
```

File: docs/PersonalApplications.rst [hunk 3]
```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
...
+
+You will receive the **News post in a space** notification when a user posts an article in a space of which you are member.
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive the **News posted in a space** notification when an user posts an article in a space you are member of.
```

File: docs/PersonalApplications.rst [hunk 4]
```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
...
+
+	|image97|
+
+**News share in a space**
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
**News shared in a space**
```

File: docs/PersonalApplications.rst [hunk 5]
```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
...
+
+You will receive the **News share in a space** notification when a user shares an article in a space of which you are member.
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive the **News shared in a space** notification when an user shares an article in a space you are member of.
```

File: docs/PersonalApplications.rst [hunk 6]
```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
...
+
+	|image98|
+
+**My News shared**
+
+You will receive the **My News shared** notification when a user shares your posted article in another space.
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
You will receive the **My News shared** notification when an user shares one of your articles in another space.
```

File: docs/PersonalApplications.rst [hunk 7]
```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
...
+
+	|image99|
+
+**My News liked**
+
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst:1289]: Missing contents ?

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/PersonalApplications.rst (hunk 7 above)
Comment (samehaidi) [reply docs/PersonalApplications.rst:1289]: These notifications are not available yet. I only listed the list of upcoming notifications.

Comment (thomasdelhomenie) [reply docs/PersonalApplications.rst:1289]: :+1:

File: docs/PersonalApplications.rst (hunk 3 above)
Comment (thomasdelhomenie) [reply docs/PersonalApplications.rst]: "A user" is actually correct (https://english.stackexchange.com/a/105117). Thanks @samehaidi  !

File: docs/PersonalApplications.rst [hunk 8]
```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
...
+You will receive the **My News shared** notification when a user shares one of your articles in another space.
+
+	|image99|
+
+**My News liked**
+
+**My News commented**
+
+**Kudos recieved for my News**
```
Comment (thomasdelhomenie) [inline docs/PersonalApplications.rst]:
```suggestion
**Kudos received for my News**
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

**Review by thomasdelhomenie — APPROVED** [review]

**Review by plamarque — COMMENTED** [review]
LGTM

**Pull request merged**